<?xml version="1.0" encoding="UTF-8"?>

<launch>
	<!--GPS-->
	<include file="$(find ublox_gps)/launch/ublox_device.launch"></include>

	<!--IMU-->
	<node pkg="myahrs_driver" type="myahrs_driver" name="myahrs_driver">
		<param name="port" value="/dev/ttyIMU" />
		<param name="baud_rate" value="115200" />
	</node>

	<!-- Nodes -->
	<node pkg="tricat231_pkg"  type="gnss_converter.py" name="gnssConverter" respawn="true"/>
	<node pkg="tricat231_pkg"  type="heading_converter.py" name="headingConverter" respawn="true"/>
	<node pkg="tricat231_pkg" type="hopping.py" name="hoppingTour" output="screen"/>
	<!-- <node pkg="tricat231_pkg" type="serial_node.py" name="ARDUINO" args="/dev/ttyARDUINO" respawn="true"/> -->

	<!-- Record
	<arg name="do_record" default="true" />
	<node pkg="rosbag" type="record" name="rosbag_record" required="true" args="record -a -O $(find tricat231)/data/rosbag/$(arg filename)" /> -->
	
	<!-- Parameters -->
	<rosparam command="load" file="$(find tricat231_pkg)/params/hopping_params.yaml"/>
	
</launch>